src/pages/settings/user/components/Details.tsx

```diff
@@ -33,36 +43,80 @@ export function Details() {
         <Card title={t('details')}>
           <Element leftSide={t('first_name')}>
             <InputField
-              id="first_name"
               value={userChanges?.first_name || user?.first_name || ''}
-              onChange={handleChange}
+              onValueChange={(value) =>
+                handleChange('first_name', value as string)
+              }
             />
           </Element>
 
           <Element leftSide={t('last_name')}>
             <InputField
-              id="last_name"
               value={userChanges?.last_name || user?.last_name || ''}
-              onChange={handleChange}
+              onValueChange={(value) =>
+                handleChange('last_name', value as string)
+              }
             />
           </Element>
 
           <Element leftSide={t('email')}>
             <InputField
-              id="email"
               value={userChanges?.email || user?.email || ''}
               type="email"
-              onChange={handleChange}
+              onValueChange={(value) => handleChange('email', value as string)}
```

Comment:
`InputField` casts the value as `string`. Not sure why we have this assertion?

Reply:
removed